src/contracts/IForwarder.json

```diff
@@ -0,0 +1,218 @@
+{
```

Comment:
i must have missed this addition until now, i'm a bit surprised we could do without it before [the original PR](https://github.com/wilsoniovlabs/relaying-services-sdk-dapp/pull/1).

any insights on this?

Reply:
I think it is used in the execute modal and it was already there in the original PR as well.